packages/salesforce/src/__tests__/recordFactory.test.ts

```diff
@@ -0,0 +1,40 @@
+import 'jest';
+
+import { RecordFactory } from '../queryRecordFactory';
+import { QueryResult } from '../queryService';
+import { SObjectRecord } from '../types/sobjectRecord';
+
+describe('recordFactory', () => {
+
+    describe('#transformDateValue', () => {
```

Comment:
```suggestion
    describe('#create', () => {
```
Usually I name the test suites after the method they are testing